src/utils/data-manager.js

```diff
@@ -332,9 +332,9 @@ export default class DataManager {
     return node;
   }
 
-  getFieldValue = (rowData, columnDef) => {
+  getFieldValue = (rowData, columnDef, inFilter) => {
```

Comment:
can you change it as (rowData, columnDef, lookup = true) , please? Because it may be used for different cases.